docs/modules/ROOT/pages/list-of-camel-quarkus-extensions.adoc

```diff
@@ -51,6 +51,9 @@ Number of Camel components: 25 in 21 JAR artifacts (0 deprecated)
 | xref:extensions/microprofile-metrics.adoc[MicroProfile Metrics] (camel-quarkus-microprofile-metrics) +
 `microprofile-metrics:metricType:metricName` | 0.2 | Camel metrics exposed with Eclipse MicroProfile Metrics
 
+| link:https://camel.apache.org/components/latest/netty-component.html[Netty] (camel-quarkus-netty) +
```

Comment:
This should be an xref pointing to the local docs in quarkus.

Reply:
> This should be an xref pointing to the local docs in quarkus.

The link or xref are auto-generated depending on whether `docs/modules/ROOT/pages/extensions/netty.adoc` exists or not. If `netty.adoc` does not exist, a link to the Camel component page is generated.

@dhartford should create `netty.adoc` only if the usage of the new netty extension differs from the stock Camel component.

Reply:
put a basic netty.adoc in place, even though just stock for now - I'm not sure if there is a good way to mark something as 'experimental', so simply added to this document location.  The camel-netty component brings an enormous range of features.